jsons/parameters.json

```diff
@@ -251,6 +251,19 @@
         },
         "value_type": "bool"
     },
+    {
+        "default_value": 8,
+        "description": "A measurement duration is allowed to exceed median duration (m) before calling it an interruption. This param defines the factor of m, which if exceeded will define a measurement interruption and therefore allow for batch separation. Default value of 8 is based on -> usually m = 2 to 2.5 min, so this is at least 15 min which is less than the minimum time it takes to clean the ion source",
```

Comment:
Factor that is used to multiple the media delta between MS runs in order to define a time threshold, which is then used to separate batches.